repos/system_upgrade/common/actors/verifytargetversion/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import verifytargetversion
+from leapp.models import Report
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class VerifyTargetVersion(Actor):
```

Comment:
let's rather use `CheckTargetVersion` instead of `Verify` to correspond with usual naming used for actors in these repositories.